lib/reel/session.rb

```diff
@@ -27,13 +27,14 @@ def self.timers_hash
     end
 
     # changing/modifying configuration
-    def self.configuration options={}
-      if @options
-        @options.merge! options if options.is_a? Hash
+    def self.configuration server, options={}
+      @options ||= {}
+      if @options[server]
+        @options[server].merge! options if Hash === options
       else
-        @options = DEFAULT_CONFIG.merge options if options.is_a? Hash
+        @options[server] = DEFAULT_CONFIG.merge options if Hash === options
       end
-      @options
+      @options[server]
     end
```

Comment:
all of this looks great, i'd like to see a test using different mock servers to make sure different configuration hashes are kept.